test/Microsoft.TestPlatform.AcceptanceTests/DifferentTestFrameworkSimpleTests.cs

```diff
@@ -117,6 +123,11 @@ public void WebTestRunAllTestsWithRunSettings(RunnerInfo runnerInfo)
     [NetFullTargetFrameworkDataSource]
     public void CodedWebTestRunAllTests(RunnerInfo runnerInfo)
     {
+        if (!IsCI)
+        {
+            Assert.Inconclusive("This works on server but not locally, because locall it grabs dll from my GAC and that is old, but has 10.0.0 version as the one in our package.");
```

Comment:
```suggestion
            Assert.Inconclusive("This works on server but not locally, because locally it grabs old dll from GAC, but has version 10.0.0 as the one in our package.");
```